src/content/index.js

```diff
@@ -2,10 +2,10 @@
 import findAndReplaceDOMText from 'findandreplacedomtext';
 import {
     getUniqueLocationsFromCurrentPage,
-} from './api';
+} from './api.js';
```

Comment:
was this giving you problems? for me, .js is the default extenstion

Reply:
It was giving me problems for some reason

Reply:
oh.. hmm ok well then we can keep it in there for now but eventually lets come back to this becuse i don't believe its common convention